xml/System.Text/Rune.xml

```diff
@@ -977,9 +977,9 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="ch" Type="System.Char" />
       </Parameters>
       <Docs>
-        <param name="ch">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ch">The 16-bit Unicode character to convert.</param>
+        <summary>Defines an explicit conversion of a 16-bit Unicode character to a <see cref="T:System.Text.Rune"/>.</summary>
```

Comment:
The summary could be improved. See this op_Explicit description: https://docs.microsoft.com/en-us/dotnet/api/system.intptr.op_explicit?view=netcore-3.0

Reply:
Same suggestion for the other op_Explicit below.

Reply:
You mean using similar words to IntPtr.Explicit summary?

I used [Decimal.Explicit](https://docs.microsoft.com/en-us/dotnet/api/system.decimal.op_explicit?view=netcore-3.0) as a base, but I can switch it if you think is better your proposal.

Reply:
I think that the language from Decimal.Explicit ("Defines an explicit conversion...") is better, given that an explicit conversion requires a casting operator -- it performs a conversion only in a qualified or conditional sense.

Reply:
No problem then, let's keep the current language.